src/producer.rs

```diff
@@ -26,7 +26,8 @@ use util::cstr_to_owned;
 
 /// A native rdkafka-sys topic, used for message production.
 struct NativeTopic {
-    ptr: *mut RDKafkaTopic,
+    client_ptr: *mut RDKafka,
```

Comment:
This is actually not really a change, I only renamed the field to make it's meaning more clear.